app/scripts/modules/google/webpack.config.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+
+const path = require('path');
+const basePath = path.join(__dirname, '..', '..', '..', '..');
+const NODE_MODULE_PATH = path.join(basePath, 'node_modules');
+const HappyPack = require('happypack');
+const happyThreadPool = HappyPack.ThreadPool({ size: 3 });
+const webpack = require('webpack');
+const exclusionPattern = /(node_modules|\.\.\/deck)/;
+
+module.exports = {
+  context: basePath,
+  entry: {
+    lib: path.join(__dirname, 'src', 'index.ts'),
+  },
+  output: {
+    path: path.join(__dirname, 'lib'),
+    filename: '[name].js',
+    library: '@spinnaker/google',
+    libraryTarget: 'umd',
+    umdNamedDefine: true,
+  },
+  externals: {
+    '@spinnaker/core': '@spinnaker/core',
+    'angular': 'angular',
+    'lodash': 'lodash',
+    'rxjs': 'rxjs',
+    '@uirouter/core': '@uirouter/core',
+    '@uirouter/angularjs': '@uirouter/angularjs',
+    'exports-loader?"n3-line-chart"!n3-charts/build/LineChart.js': 'exports-loader?"n3-line-chart"!n3-charts/build/LineChart.js',
```

Comment:
You don't need this one, since the google module doesn't use the n3-line-chart library.